### PR TITLE
initial code - application-manager

### PR DESCRIPTION
**Pull request by seanhuxy**
fixes https://github.com/vmware/dispatch/issues/90

### CODE REVIEW

File: charts/dispatch/charts/application-manager/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        paths:
+          - path: {{ .Values.ingress.path }}
+            backend:
+              serviceName: {{ include "fullname" . }}
+              servicePort: {{ .Values.service.externalPort }}
+  {{- if $tls.secretName }}
+  tls:
+    - secretName: {{ $tls.secretName }}
+      hosts:
+        - {{ default .Values.global.host .Values.ingress.host }}
```
Comment (berndtj) [inline charts/dispatch/charts/application-manager/templates/ingress.yaml]: We/I changed the templates for how the hosts/ingress is defined.  We should be consistent.  Which could mean making all the other charts more "standard" or making this chart like the others.

Comment (seanhuxy) [reply charts/dispatch/charts/application-manager/templates/ingress.yaml]: I will double check

File: charts/dispatch/charts/application-manager/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{{- if .Values.ingress.enabled -}}
+{{- $tls := default .Values.global.tls .Values.ingress.tls -}}
```
Comment (neosab) [inline charts/dispatch/charts/application-manager/templates/ingress.yaml]: `{{- $ingress_host := default .Values.global.host .Values.ingress.host -}}`

File: charts/dispatch/charts/application-manager/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    ingress.kubernetes.io/auth-signin: https://$host/oauth2/start
+    ingress.kubernetes.io/auth-url: http://{{ .Release.Name }}-oauth2-proxy.{{ .Release.Namespace }}.svc.cluster.local/oauth2/auth
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+    - host: {{ default .Values.global.host .Values.ingress.host }}
```
Comment (neosab) [inline charts/dispatch/charts/application-manager/templates/ingress.yaml]: Can you add this check here:- 
      ```
{{- if $ingress_host }}
host: {{ $ingress_host }}
{{- end -}}
```

**Comment by neosab** [comment]
Just a minor change and I am fine with this patch :)

**Pull request merged**